client/packages/system/src/Manage/Preferences/api/useAdminPrefsList.ts

```diff
@@ -2,15 +2,21 @@ import { PreferenceNodeType, useQuery } from '@openmsupply-client/common';
 import { usePreferencesGraphQL } from './usePreferencesGraphQL';
 import { PREFERENCE_DESCRIPTIONS } from './keys';
 
-export const useAdminPrefsList = (prefType: PreferenceNodeType) => {
-  const { api, storeId } = usePreferencesGraphQL();
+export const useAdminPrefsList = (
+  prefType: PreferenceNodeType,
+  storeId?: string
+) => {
+  const { api, storeId: loggedInStoreId } = usePreferencesGraphQL();
 
   return useQuery({
-    queryKey: [PREFERENCE_DESCRIPTIONS, prefType],
+    queryKey: [PREFERENCE_DESCRIPTIONS, prefType, storeId],
     queryFn: async () => {
       const result = await api.adminPreferenceList({
-        storeId,
+        storeId: loggedInStoreId,
         prefType,
+        context: {
+          storeId,
```

Comment:
When querying prefType = STORE, should include the store ID of the store whose prefs we want to load